packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -273,6 +277,11 @@ class TimelineHttpRequestData extends HttpRequestData {
   final TraceEvent _startEvent;
   TraceEvent _endEvent;
 
+  /// For timeline event, HTTP request/response body logging is disabled.
```

Comment:
will you add this same comment to responseBody below since it also applies to responseBody? Also, you can set equal to null there as well.

Reply:
I wanted to do so :-) , but responseBody is initialised in TimelineHttpRequest._ constructor. and this constructor is called from 
TimelineHttpRequestData.fromTimeline factory.

test/support/network_test_data.dart calls this factory to initialise fake data for several tests. And some of those tests check that responseBody is not null.

TimelineHttpRequest seems to be obsolete since dartIOVersion 1.6, but does it make sense to update those fake data in tests ? Is it supported to use new DevTools build with old Dart SDK ? If yes we should keep those tests and add new ones.

Reply:
Ah I see. Can you file an issue to cleanup the TimelineHttp code to default to the post dartIOVersion 1.6 world? If you can link to these comments and assign to me, that would be great.